bitnami/redis-cluster/templates/metrics-prometheus.yaml

```diff
@@ -7,18 +7,33 @@ metadata:
   namespace: {{ .Values.metrics.serviceMonitor.namespace }}
   {{- end }}
   labels: {{- include "common.labels.standard" . | nindent 4 }}
+    {{- if .Values.metrics.serviceMonitor.selector }}
+    {{- include "common.tplvalues.render" (dict "value" .Values.metrics.serviceMonitor.selector "context" $) | nindent 4 }}
+    {{- end }}
```

Comment:
The `.Values.metrics.serviceMonitor.selector` should NOT be applied here but in the "selector" field (line 37)